src/event_sources/ble_esp32.cc

```diff
@@ -79,8 +79,6 @@ void BLEEventSource::entry() {
 
   while (!_stop) {
     if (_should_run) {
-      nimble_port_init();
```

Comment:
I'd prefer keeping the nimble_port_init call here, because of the symmetry with the deinit call below. Is there a reason you moved this?

Reply:
I see your point, however, unfortunately, the gatt server setup requires nimble_port_init to be called before it sets up its resources (init_server). I'll add a comment.

Reply:
Actually, looking at it, the nimble_port_freertos_deinit is not needed at all, as we are not running the freertos task, but our own thread. So, I'll go ahead and remote the deinit call and symmetry is restored